packages/react/error-boundary/src/ErrorBoundary.tsx

```diff
@@ -77,7 +75,7 @@ class BaseErrorBoundary extends Component<ErrorBoundaryProps, ErrorBoundaryState
     this.resetState();
   };
 
-  componentDidUpdate(prevProps: ErrorBoundaryProps) {
+  componentDidUpdate(prevProps: Props) {
```

Comment:
I want to leave comment with this link

https://github.com/bvaughn/react-error-boundary/pull/152#discussion_r1272742523

Reply:
I know what you consider but this is not necessary in our case.